core/mail_api.php

```diff
@@ -1332,6 +1343,13 @@ private function parse_from_field( $p_from_address )
 		return( $v_from_address );
 	}
 
+	# --------------------
+	# Return the date from the mail's 'Date' field
+	private function parse_date_field( $p_date )
+	{
+		return strtotime($p_date);
```

Comment:
This function often uses the system locale as a base for some decisions. Since emails can come from all around the world those decisions could be made incorrectly